src/tests/test_targets.py

```diff
@@ -112,5 +117,10 @@ def test_destroy_available_terraform(
                     controller.tf.set_vars(**config_vars)
                     controller.destroy_all_nodes()
                     destroyed_clusters += 1
+                    resources_deleted = True
```

Comment:
Do we need this boolean? Or we can detect if deletion occurred by checking if the files exist or not

Reply:
We want tot delete the resource folder only if the resources were destroyed, because that this target destroy specific platform resources a folder could exist but resources wasn't destroyed yet